config/routes.rb

```diff
@@ -3,9 +3,10 @@
 
   # Defines the root path route ("/")
   root "home#index"
-  resources :forms, only: [:index] do
-    resource :page, module: 'forms', only: [:show, :create], path: "/" do
-      get :submitted
-    end
-  end
+
+  resources :form, only: [], path: '/' do
+   resource :page, only: [:new, :create], module: 'forms', path_names: { new: '/' }, path: '/' do
+     get :submitted
+   end
+ end
```

Comment:
Looks like the indentation has gone a bit weird here - think it should be:
```suggestion
  resources :form, only: [], path: '/' do
    resource :page, only: [:new, :create], module: 'forms', path_names: { new: '/' }, path: '/' do
      get :submitted
    end
  end
```